lib/eco/inclexer/inclexer.py

```diff
@@ -515,8 +515,10 @@ def relex(self, node):
             try:
                 token = next_token()
                 lookaheads.append(token[2])
-                if token[3][0] is startnode:
-                    past_startnode = True
+                for temp in token[3]:
```

Comment:
Is it just me or is this loop only worth taking if `past_startnode` is `False`? [I don't know if the loop is over a long data-structure or not; if it is potentially long, adding `if not past_startnode` might be a useful optimisation.]